app/views/annual_evaluations/_form.html.slim

```diff
@@ -1,33 +1,44 @@
 = wrapped_form_for([@patient, @annual_evaluation]) do |f|
   = f.error_notification
-
   .form-inputs
-    h2 Clinical
-    = f.input :bmi
-    = f.input :bladder_drainage_method, collection: AnnualEvaluation.bladder_drainage_methods.keys
+    h4 Clinical
+    = f.input :bmi, label: "BMI", :placeholder => "00"
+    = f.input :bladder_drainage_method,  label: "Bladder Drainage Method", :prompt => "Please select one", collection: AnnualEvaluation.bladder_drainage_methods.keys
     = f.input :has_motor_or_sensory_asymmetry
-    = f.input :neuro_zone_of_preservation1, collection: AnnualEvaluation.neuro_zone_of_preservation1s.keys
-    = f.input :neuro_zone_of_preservation2, collection: AnnualEvaluation.neuro_zone_of_preservation2s.keys
-    = f.input :neuro_zone_of_preservation3
+    = f.input :neuro_zone_of_preservation1,  label: "Neuro Zone of Preservation", :prompt => "Please select one", collection: AnnualEvaluation.neuro_zone_of_preservation1s.keys
+    = f.input :neuro_zone_of_preservation2, label: false, :prompt => "Please select one", collection: AnnualEvaluation.neuro_zone_of_preservation2s.keys
+    = f.input :neuro_zone_of_preservation3, label: false, :placeholder => "00"
 
-    h2 ASIA and LOI
-    = f.input :level_of_injury
-    = f.input :asia_score
+    h4 ASIA &mdash; LOI
+    .instrument
+        h4
+          | ASIA-LOI: A-C6  
+          span.small = link_to "view record: 12/23/1987"
+        h4
+            = link_to "+ add new assessment", new_patient_omr1_year_path(@patient)
+        / = f.input :level_of_injury
+        / = f.input :asia_score
 
-    h2 Administrative
-    = f.input :eval_offered, label: "Date Offered", as: :string, input_html: { class: "datepicker" }
-    = f.input :eval_completed, label: "Date of Completion", as: :string, input_html: { class: "datepicker" }
-    Display next AE Due here.
+    h4 Administrative
+    = f.input :eval_offered, label: "Date Offered", :placeholder => "June 10, 2015", as: :string, input_html: { class: "datepicker" }
```

Comment:
Should we use the current date for a place holder?  And does this understand the date format June 10, 2015? I thought we needed mm/dd/yyyy